on-prem-docs/README.MD

```diff
@@ -0,0 +1,75 @@
+# Chef Habitat Builder on-prem  - Website
+
+A new static website for Chef Habitat on-prem is based on [Hugo](https://gohugo.io), the Go-powered static site generator.
+
+## Getting Started
+
+### One Time Install
+
+1. Disable smart quotes. OS X tries to use smartquotes by default when editing Hugo docs. You want smartquotes disabled regardless if you are writing code. Here's how [you can disable it](http://www.iclarified.com/38772/how-to-disable-curly-quotes-in-mac-os-x-mavericks).
+
+1. [Install Hugo](https://gohugo.io/getting-started/installing/). If you're using homebrew:
+
+    ```
+    brew install hugo
+    ```
+
+### View the docs
+
+From the `on-prem-docs` directory, run `hugo server` to serve the docs.
+Use `hugo server -D` to serve the docs with draft content
+
+### Development
+
+1. From the `on-prem-docs` directory, run the following command to start the development server.
+
+    ```
+    $ make serve
+    ```
+
+    When this is running, any changes you make to content in the `components/automate-chef-io` directory will be automatically compiled and updated the browser.
+
+## Creating Blog Posts and Making Docs Changes
```

Comment:
We should remove reference to Blog Posts

Reply:
done